dom-manipulation/main.ts

```diff
@@ -0,0 +1,29 @@
+let $clickAmount = 0;
```

Comment:
This isn't storing a dom element, so it shouldn't have a dollar sign in front. It should just be `clickAmount`